packages/delivery-react-native/delivery.js

```diff
@@ -29,7 +29,8 @@ module.exports = (client, NativeClient) => ({
       groupingHash: event.groupingHash,
       apiKey: event.apiKey,
       featureFlags: event.toJSON().featureFlags,
-      nativeStack: nativeStack
+      nativeStack: nativeStack,
+      correlation: event.toJSON().correlation
```

Comment:
Do we need to call `toJSON` here? It looks like we could read directly from `event._correlation` instead

Reply:
I had started with that, but for some reason that was always `undefined` in my initial testing, but I may have conflated it with another issue I was seeing. Unit tests still passing with `event._correlation` so that's what we'll do :)